src/main/java/command/ShowAfterCommand.java

```diff
@@ -0,0 +1,71 @@
+package command;
+
+import java.time.LocalDate;
+
+import exception.DukeDateTimeParserException;
+import parser.Parser;
+import storage.Storage;
+import task.DeadlineTask;
+import task.EventTask;
+import task.Task;
+import task.TaskList;
+import ui.Ui;
+
+
+/**
+ * ShowAfterCommand would execute the program when user specify
+ * "show after" as the command. This would show all the task that exist
+ * after the specified date that was passed in the user command.
+ */
+public class ShowAfterCommand extends Command {
+
+    private String command;
+
+    /**
+     * Constructs a ShowAfterCommand with the given
+     * user command.
+     *
+     * @param command String user command
+     */
+    public ShowAfterCommand(String command) {
+        super();
+        this.command = command;
+    }
+
+    /**
+     * Executes parsed user command. The result is:
+     * 1. Shows user all the task that exist after the specified date.
+     *
+     * @param tasks TaskList List of task.
+     * @param ui Ui updating user interface to show intended messages.
+     * @param storage Storage to update external file whenever needed.
+     * @throws DukeDateTimeParserException This exception would be thrown
+     * when the user failed to specify the date in the command.
+     */
+    @Override
+    public void execute(TaskList tasks, Ui ui, Storage storage) throws DukeDateTimeParserException {
+        LocalDate localDate = Parser.findDateParser(this.command);
+        StringBuilder sb = new StringBuilder();
+        int i = 1;
+
+        for (Task task : tasks.getTasks()) {
+            if (task instanceof DeadlineTask) {
+                DeadlineTask deadlineTask = (DeadlineTask) task;
+                if (deadlineTask.getDateTime().toLocalDate().isAfter(localDate)) {
+                    sb.append(ui.formatMessage(i + ". " + deadlineTask + "\n"));
+                    i++;
+                }
+
+            } else if (task instanceof EventTask) {
+                EventTask eventTask = (EventTask) task;
+                if (eventTask.getDateTime().toLocalDate().isAfter(localDate)) {
```

Comment:
This seems like another possible case of breaking abstraction. Consider including a method for the two types of `Tasks` to make this check themselves.